com.github.corna.Vivado.yml

```diff
@@ -0,0 +1,93 @@
+app-id: com.github.corna.Vivado
+runtime: org.gnome.Sdk
+runtime-version: '40'
+sdk: org.gnome.Sdk
```

Comment:
does it really need the GNOME runtime?

If all you need is Gtk3 then it's in the freedesktop-sdk

Reply:
I used freedesktop.Sdk previously, however Vitis (eclipse-based) fails to start with the freedesktop one on X11 (see commit 7069d96e1cfa0c3a6bfde8ef5d8d77819a20998a). I haven't yet understood why though.

Reply:
ok